code/__HELPERS/traits.dm

```diff
@@ -139,6 +139,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_BLOODCRAWL_EAT	"bloodcrawl_eat"
 #define TRAIT_DWARF				"dwarf"
 #define TRAIT_SILENT_FOOTSTEPS	"silent_footsteps" //makes your footsteps completely silent
+#define TRAIT_MESON_VISION		"meson_vision"
+#define TRAIT_FLASH_PROTECTION 	"flash_protection"
+#define TRAIT_NIGHT_VISION 		"night_vision"
```

Comment:
You've got some spaces here which can be converted into tabs:
![image](https://user-images.githubusercontent.com/57483089/127349066-fddfad89-782c-4aa1-af22-0a081f2022f7.png)
![image](https://user-images.githubusercontent.com/57483089/127349011-ff915572-77cb-4b47-b202-0fd08fc284b3.png)
```suggestion
#define TRAIT_MESON_VISION		"meson_vision"
#define TRAIT_FLASH_PROTECTION	"flash_protection"
#define TRAIT_NIGHT_VISION		"night_vision"
```